### DISCUSSION TITLE
Import/export configuration data from/to the store

### DISCUSSION THREAD

**Discussion by baterfly85** [openfga/community#141] [Q&A]
Is there a programmatic way to import and export configuration data from the FGA store? Some sort of CLI or API that allows doing that similar to [Auth0 CLI export&import commands](https://auth0.com/docs/deploy-monitor/deploy-cli-tool/using-as-a-cli)? 

If not, what could be the best way to do this?

**Comment by jon-whit** [comment]
👋 @baterfly85 

I'm not sure exactly what you mean by "configuration data", but if you are referring to authorization models, assertions, and relationship tuples, then you can push them in and fetch them programmatically through the [API](https://openfga.dev/api/service). However, we have plans to work on an official OpenFGA CLI, and we will have features to import/export these things through the CLI as well. 

Keep an eye on https://github.com/openfga/roadmap/issues/17 for updates.
